docs/docs/ecommerce-tutorial/index.md

```diff
@@ -15,7 +15,15 @@ title: "Gatsby E-Commerce Tutorial"
   - [Getting your Stripe test keys](#getting-your-stripe-test-keys)
 - [Examples](#examples)
   - [Easy: One Button](#easy-one-button)
+    - [Create a product and SKU](#create-a-product-and-sku)
```

Comment:
So once-more here, I think I like this, but this could be a bit too granular for a tutorial?

Reply:
I can't figure out where these changes came from. It doesn't really seem like this could have been generated. I'll work through the diff and get rid of them all.